blog/03-28-2023-Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration/Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration.md

```diff
@@ -0,0 +1,408 @@
+---
+title: Blockchain-Based Festival Booking System with CUSD Payment Integration
+description: A decentralized platform for event organizers to showcase their programs and allow users to book their slots using cUSD. With Festival, event organizers can write descriptions of their programs including URLs, themes, descriptions, locations, and prices. Users can then browse and book their slots using cUSD tokens.
+authors:
+  - name: David Ikanji
+    title: Technical Writer 
+    url:  https://github.com/Ikanji201
+    image_url:  https://avatars.githubusercontent.com/u/115812158?v=4
+tags: [solidity, intermediate, celo, celosage]
+hide_table_of_contents: true
+slug: /tutorials/blockchain-based-festival-booking-system-with-cusd-payment-integration
+---
+
+![Blockchain-Based Festival Booking System with CUSD Payment Integration](https://user-images.githubusercontent.com/115812158/228259160-bca6cc82-4d0e-4de8-9ea0-569dc9f9e2ec.png)
```

Comment:
This is not the right format for header image, you should import it it from showcase folder